newIDE/app/src/UI/Theme/CreateTheme.js

```diff
@@ -177,6 +177,19 @@ export function getMuiOverrides(
         padding: 0,
       },
     },
+    MuiAccordionDetails: {
+      root: {
+        padding: 8,
+      },
+    },
+    // Avoid double border between buttons in ButtonGroup
+    MuiButtonGroup: {
```

Comment:
ButtonGroup is already used in `RaisedButtonWithSplitMenu`. Can you check if this change is not having bad side effects there?

Reply:
My bad - yes, this took away the border between the main button and dropdown arrow in `RaisedButtonWithSplitMenu`. Turns out the double border was something caused by passing `variant="contained"` to ButtonGroup (I think I forgot to remove it after the first mockup). I've removed that prop and this theme override, and everything works okay now.

By the way, if you enable two consecutive bits there is no border between them. A bit hard (or verbose) to fix because MUI doesn't provide a lot of help with mixed button types. I hope that's not a major problem.